virtualpdu/main.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import print_function
```

Comment:
This is not needed. Print as a function is supported since py26 and I don't think we will be going that far back.